src/tests/pages/application.pages.js

```diff
@@ -0,0 +1,18 @@
+export class ApplicationPages {
+    constructor(page) {
+        this.url = "/";
+        this.page = page;
+        this.forTeachersMenuItemLocator = page.getByRole("button", { name:" Pro učitelé" });
+        this.newOrderLocator = page.getByRole("link", { name: "Objednávka pro MŠ/ZŠ"});
+    //    this.newOrderPage = new OrdersPage(this.page);
+    }
+
+    async open() {
+        await this.page.goto(this.url);
+    }
+
+    async gotoNewOrder() {
```

Comment:
_Goto_ a _navigate_ jsou typicky používané pro navigaci přes url. V rámci konzistence bych změnil třeba na _open_.
```suggestion
    async openNewOrder() {
```